packit_service/worker/helpers/testing_farm.py

```diff
@@ -60,40 +61,72 @@ class CommentArguments:
     Parse arguments from trigger comment and provide the attributes to Testing Farm helper.
     """
 
-    packit_command: str = None
-    identifier: str = None
-    labels: List[str] = None
-    pr_argument: str = None
-
     def __init__(self, command_prefix: str, comment: str):
```

Comment:
the `__init__` method grew quite a lot with the new code, do you think it could be split up a bit?

Reply:
Sure, I will try to split it